### PR TITLE
Login vernon new Pull Request with fixes and fixes to aforementioned fixes

### PR DESCRIPTION
**Pull request by VernBuck**

### CODE REVIEW

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 1]
```diff
@@ -12,8 +13,6 @@
 import amazingme.app.EnumeratedActivity;
 import amazingme.model.AmazingMeAppCompatActivity;
 
-import static com.amazingme.activities.R.id.user_profile_back_button;
-
 public class ChildRegistrationActivity extends AmazingMeAppCompatActivity {
 
     //private Button backBtn, doneBtn;
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: is there a reason you commented most of this file out?

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 2]
```diff
@@ -23,6 +22,7 @@
     private EditText firstnameEditText, lastnameEditText, ageEditText, sexEditText;
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: camelCase the appropriate variables in this file, please :)

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 3]
```diff
@@ -43,29 +43,30 @@ public void bindToUserInterface() {
...
         //@Override
         //public void onClick(View v) {
-        backBtn = (Button) findViewById(user_profile_back_button);
-        doneBtn = (Button) findViewById(R.id.button3);
-        childBtn = (Button) findViewById(R.id.button2);
-        firstnameEditText = (EditText) findViewById(R.id.editText2);
-        lastnameEditText = (EditText) findViewById(R.id.editText4);
-        ageEditText = (EditText) findViewById(R.id.editText5);
+        profilepic = (QuickContactBadge) findViewById(R.id.child_registration_child_portrait);
+        backBtn = (Button) findViewById(R.id.child_registration_back_button);
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: any reasons you commented out methods and put everything right here?

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 4]
```diff
@@ -43,29 +43,30 @@ public void bindToUserInterface() {
...
+        doneBtn = (Button) findViewById(R.id.child_registration_done_button);
+        childBtn = (Button) findViewById(R.id.child_registration_add_another_child);
+        firstnameEditText = (EditText) findViewById(R.id.child_registration_first_name);
+        lastnameEditText = (EditText) findViewById(R.id.child_registration_last_name);
+        ageEditText = (EditText) findViewById(R.id.child_registration_date_of_birth);
         //sexEditText = (EditText) findViewById(R.id.editText7);
-        profview = (TextView) findViewById(R.id.textView4);
-        abilityview = (TextView) findViewById(R.id.textView5);
-        sexview = (TextView) findViewById(R.id.lable);
+        profview = (TextView) findViewById(R.id.user_profile_parent_profile_label);
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: should be child_registration not user_profile

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 5]
```diff
@@ -43,29 +43,30 @@ public void bindToUserInterface() {
...
+        abilityview = (TextView) findViewById(R.id.child_registration_question_label);
+        sexview = (TextView) findViewById(R.id.child_registration_sex_label);
 
-        yesnoSpinner = (Spinner) findViewById(R.id.spinner1);
-        malefemale = (Spinner) findViewById(R.id.spinner2);
+        yesnoSpinner = (Spinner) findViewById(R.id.child_registration_disability_dropdown);
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: not yes or no. it'll be a list.

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 6]
```diff
@@ -43,29 +43,30 @@ public void bindToUserInterface() {
...
+        malefemale = (Spinner) findViewById(R.id.child_registration_sex_dropdown);
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: let's just call this the sexSpinner. it sounds goofy but whatever it's what it is.

File: app/src/main/java/amazingme/activities/app/GameSelectionPage.java [hunk 7]
```diff
@@ -3,6 +3,7 @@
 import android.view.View;
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/GameSelectionPage.java]: all of this is fine for now... be ready to pick it back up (we need to dynamically load the page based on what games are available).

File: app/src/main/java/amazingme/activities/app/LoginActivity.java [hunk 8]
```diff
@@ -50,8 +50,7 @@ public void onClick(View view) {
             public void onClick(View view) {
                 final String email = emailEditText.getText().toString();
                 final String password = passwordEditText.getText().toString();
-                goTo(EnumeratedActivity.CHILD_REGISTRATION);
-                //AmazingMeApplicationContext.restoreSession(email, password, LoginActivity.this);
+                goTo(EnumeratedActivity.GAME_SELECTION_PAGE);
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/LoginActivity.java]: why did you change this? 

definitely needs to be put back. it's fine if it's for testing, but make sure you put it back.

File: app/src/main/java/amazingme/activities/app/UserProfileActivity.java [hunk 9]
```diff
@@ -28,7 +32,8 @@ public void bindToUserInterface() {
         nextBtn = (Button) findViewById(R.id.user_profile_next_button);
         firstName = (EditText) findViewById(R.id.user_profile_first_name_edit_text);
         lastName = (EditText) findViewById(R.id.user_profile_last_name_edit_text);
-        parentEmail = (EditText) findViewById(R.id.user_profile_parent_email_edit_text);
+        label = (TextView) findViewById(R.id.user_profile_parent_profile_label);
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/UserProfileActivity.java]: i don't think we need the label in this class....?

File: app/src/main/java/amazingme/activities/app/UserProfileActivity.java [hunk 10]
```diff
@@ -28,7 +32,8 @@ public void bindToUserInterface() {
...
+        profile = (QuickContactBadge) findViewById(R.id.user_profile_parent_portrait);
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/UserProfileActivity.java]: we need to see if this is what we need for a picture... 

we need to be able to set it and i'm not sure if this is what we will use... but let's keep it for now. good job putting it in there!

File: app/src/main/res/layout/activity_child_registration.xml [hunk 11]
```diff
@@ -8,54 +8,55 @@
     android:background="@drawable/upperbackgroundlb">
```
Comment (zalaby12) [inline app/src/main/res/layout/activity_child_registration.xml]: can we do something about the way it looks? 

i don't care if we have to switch things out it just looks... sloppy still. like everything is really spread out and nothing sticks out. 

reference the other amazing me material for some inspiration.

File: app/src/main/res/layout/activity_game_selection_page.xml [hunk 12]
```diff
@@ -7,16 +7,20 @@
     tools:context="amazingme.activities.app.GameSelectionPage">
 
     <ImageView
```
Comment (zalaby12) [inline app/src/main/res/layout/activity_game_selection_page.xml]: pretty sure we are gonna have to make this a bunch of containers on the page. 

because we are loading it dynamically, we need to be able to put each game's preview and button into the container and then populate the number of containers.... 

see if we can't consolidate this into the upper left corner or something.

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java (hunk 1 above)
Comment (VernBuck) [reply app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: I wasn't sure whether the code would be needed for backend or not so I chose to comment it rather than erase

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java (hunk 3 above)
Comment (VernBuck) [reply app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java]: all comments on the page were either for code I was unsure of how to use or for something that I thought could be useful later on

**Comment by VernBuck** [comment]
new commits with updates req. review

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 13]
```diff
@@ -1,31 +1,32 @@
...
 
 public class ChildRegistrationActivity extends AmazingMeAppCompatActivity {
 
-    private Button backBtn, doneBtn;
-    private Spinner birthMonth, birthDay, birthYear;
-    private String month, day, year;
-
-    public ChildRegistrationActivity() { super(R.layout.activity_child_registration); }
+    //private Button backBtn, doneBtn;
+    //private Spinner birthMonth, birthDay, birthYear;
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java:19]: still would love to know why you've commented this out..

File: app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java [hunk 14]
```diff
@@ -34,20 +35,55 @@ public EnumeratedActivity activityName() {
 
     @Override
     public void bindToUserInterface() {
-        initMonthSpinner();
-        initDaySpinner();
-        initYearSpinner();
-        initDoneButton();
-        backBtn = (Button) findViewById(R.id.child_registration_add_another_button);
+        //initMonthSpinner();
+        //initDaySpinner();
```
Comment (zalaby12) [inline app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java:39]: also still don't know why this was moved...

**Review by zalaby12 — APPROVED** [review]
